src/app/core/base_config.yaml

```diff
@@ -0,0 +1,98 @@
+# $ python -m nwm_routing -f -V4 test_rfc.yaml
+#--------------------------------------------------------------------------------
+log_parameters:
+    #----------
+    showtiming: True
+    log_level : DEBUG
+#--------------------------------------------------------------------------------
+network_topology_parameters:
+    #----------
+    supernetwork_parameters:
+        #----------
+        network_type: HYFeaturesNetwork
+        geo_file_path: ???
+        columns: 
+            key: 'id'
+            downstream: 'toid'
+            dx : 'length_m'
+            n : 'n'
+            ncc : 'nCC'
+            s0 : 'So'
+            bw : 'BtmWdth'
+            waterbody : 'rl_NHDWaterbodyComID'
+            gages : 'rl_gages'
+            tw : 'TopWdth'
+            twcc : 'TopWdthCC'
+            musk : 'MusK'
+            musx : 'MusX'
+            cs : 'ChSlp'
+            alt: 'alt'
+            mainstem: 'mainstem'
+    waterbody_parameters:
+        #----------
+        break_network_at_waterbodies: True  
+#--------------------------------------------------------------------------------
+compute_parameters:
+    #----------
+    parallel_compute_method: by-subnetwork-jit-clustered # serial
+    compute_kernel         : V02-structured
+    assume_short_ts        : True
```

Comment:
It could be helpful to at least point out how the API could be expanded to provide mechanics to expose other configuration variables to be manipulated by the API, which may help the maintainers unfamiliar with fastapi see what long term maintenance and enahncements may look like